app/screens/redeem-lnurl-withdrawal-screen/redeem-bitcoin-detail-screen.tsx

```diff
@@ -307,104 +291,46 @@ const RedeemBitcoinDetailScreen = ({
         </Text>
         <View style={styles.currencyInputContainer}>
           <View style={styles.currencyInput}>
-            {amountCurrency === "BTC" && (
-              <>
-                <FakeCurrencyInput
-                  value={satAmount}
-                  onChangeValue={(newValue) => setSatAmount(Number(newValue))}
-                  prefix=""
-                  delimiter=","
-                  separator="."
-                  precision={0}
-                  suffix=" sats"
-                  minValue={minWithdrawableSatoshis}
-                  style={styles.walletBalanceInput}
-                  editable={maxWithdrawable !== minWithdrawable}
-                  autoFocus
-                />
-                {minWithdrawable !== maxWithdrawable && (
-                  <Text
-                    style={
-                      satAmount <= maxWithdrawableSatoshis
-                        ? styles.infoText
-                        : styles.withdrawalErrorText
-                    }
-                  >
-                    {LL.RedeemBitcoinScreen.minMaxRange({
-                      minimumAmount: minWithdrawableSatoshis.toString(),
-                      maximumAmount: maxWithdrawableSatoshis.toString(),
-                      currencyTicker: "sats",
-                    })}
-                  </Text>
-                )}
-                <FakeCurrencyInput
-                  value={satAmountInUsd}
-                  prefix={fiatSymbol}
-                  delimiter=","
-                  separator="."
-                  precision={2}
-                  minValue={minSatAmountInUsd}
-                  maxValue={maxSatAmountInUsd}
-                  editable={false}
-                  style={styles.convertedAmountText}
-                />
-              </>
+            <MoneyAmountInput
+              moneyAmount={unitOfAccountAmount}
+              setAmount={(amount) => {
+                setUnitOfAccountAmount(amount)
+              }}
+              style={styles.walletBalanceInput}
+              editable={canSetAmount}
+            />
+            {canSetAmount && (
```

Comment:
I think this could be another PR, but the message here is not very clear imo. we got some feedback on this.

I think typically the min max is either:
- equal
- very broad (ie: 1 sats to 1 btc)

if this is equal, then we don't need to display it which is what this branch is about I think

if this is not equal, then I think we should do like when handle the balance. we only show it when we're out of bound.

Reply:
ohhh. this is for lnurl withdrawal. I thought this was for lnurl-pay. not sure if my comment is still relevant.

Reply:
I think it may be more relevant to lnurlpay. Especially if you want to try and withdraw the max amount you need to be able to see what it is.